src/binary/ibc-setup/commands/balances.spec.ts

```diff
@@ -0,0 +1,136 @@
+import fs from 'fs';
+import os from 'os';
+
+import test from 'ava';
+import sinon from 'sinon';
+
+import { IbcClient } from '../../../lib/ibcclient';
+import { generateMnemonic } from '../utils/generate-mnemonic';
+
+import { Options, run } from './balances';
+
+const consoleLog = sinon.stub(console, 'log');
+const fsReadFileSync = sinon.stub(fs, 'readFileSync');
+const ibcClient = sinon.stub(IbcClient, 'connectWithSigner');
+const mnemonic = generateMnemonic();
+
+function fakeBalance(amount: string) {
+  return (Promise.resolve({
+    query: {
+      bank: {
+        unverified: {
+          balance: sinon.fake.returns({ amount, denom: 'sampledenom' }),
+        },
+      },
+    },
+  }) as unknown) as Promise<IbcClient>;
```

Comment:
Casting is required to fulfill the huge `IbcClient` interface.

Reply:
👍 Do whatever you need in test files.